MediaBrowser.Model/Branding/BrandOptionsDto.cs

```diff
@@ -0,0 +1,25 @@
+namespace MediaBrowser.Model.Branding;
+
+/// <summary>
+/// The branding options DTO for API use.
+/// This DTO excludes SplashscreenLocation to prevent it from being updated via API.
+/// </summary>
+public class BrandOptionsDto
```

Comment:
For naming consistency we'd probably want to name this "BrandingOptionsDto". No strong preference on either option though.